Sources/HashableExtensions.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  HashableExtensions.swift
+//  EZSwiftExtensions
+//
+//  Created by Rugen Heidbuchel on 01/08/2018.
+//  Copyright © 2018 Goktug Yilmaz. All rights reserved.
+//
+
+import Foundation
+
+extension Hashable {
+    
+    /// EZSE: A string representation of the hash value.
+    var hashString: String {
```

Comment:
This is a minor thing, but can this go into StringExtensions? It makes sense to have a Hashable extension class if there are disparate types you want to deem hashable.

Reply:
Lets also add a CHANGELOG entry for this.

Reply:
I think it's better to keep it as a `Hashable` extension. If people create their own `Hashable` classes, this will also work. It will also work for other Apple classes, like `NSSet` for example. I will add a CHANGELOG entry already.

Reply:
I added the CHANGELOG entry.